ui/app/models/pki/action.js

```diff
@@ -52,7 +52,7 @@ export default class PkiActionModel extends Model {
     label: 'Subject Alternative Names (SANs)',
     editType: 'stringArray',
   })
-  altNames; // comma sep strings
```

Comment:
this seemed confusing since it's a stringArray edit type. technically the api can accept a comma separated string OR array of strings, but in the UI this renders as separate string list inputs.